terrain/fixed_lod/voxel_terrain.cpp

```diff
@@ -918,6 +926,23 @@ void VoxelTerrain::emit_data_block_unloaded(Vector3i bpos) {
 	emit_signal(VoxelStringNames::get_singleton().block_unloaded, bpos);
 }
 
+void VoxelTerrain::emit_data_mesh_block_entered(Vector3i bpos) {
```

Comment:
Should be named `emit_mesh_block_entered`. "data" was part of "data_block", here this is a "mesh_block". I think the comment also doesnt need to be copied.